commands/slash/playnext.js

```
@@ -0,0 +1,192 @@
+        .catch(this.warn);
+    }
+
+    if (res.loadType === "TRACK_LOADED" || res.loadType === "SEARCH_RESULT") {
+      player.queue.add(res.tracks[0],0);
+
+      if (!player.playing && !player.paused && !player.queue.size) {
+        player.play();
+      }
+      
+      var title = escapeMarkdown(res.tracks[0].title);
+      var title = title.replace(/\]/g, "");
```

Biome (1.9.4)

[error] Shouldn't redeclare 'title'. Consider to delete it or rename it.

'title' is defined here:


(lint/suspicious/noRedeclare)